FEATURES.md

```diff
@@ -0,0 +1,182 @@
+
+#### Details
+- **Monitor list**: Retrieve the list of monitors and their metada, including which one is primary is applicable
+
+### Input handling
+|Feature                                 |Windows|MacOS |Linux x11|Linux Wayland|Android|iOS    |Emscripten|
+|--------------------------------------- | ----- | ---- | ------- | ----------- | ----- | ----- | -------- |
+|Mouse events                            |✔️    |▢#63  |✔️       |✔️          |**N/A**|**N/A**|✔️       |
+|Mouse set location                      |✔️    |✔️    |✔️       |❓           |**N/A**|**N/A**|**N/A**  |
+|Cursor grab                             |✔️    |▢#165 |▢#242    |❌#306      |**N/A**|**N/A**|✔️       |
+|Cursor icon                             |✔️    |✔️    |✔️       |❌#306      |**N/A**|**N/A**|❌       |
+|Touch events                            |✔️    |❌    |✔️       |✔️          |✔️    |✔️     |✔️       |
+|Multitouch                              |❓     |❌    |✔️       |✔️          |❓     |❌     |❌       |
+|Keyboard events                         |✔️    |✔️    |✔️       |✔️          |❓     |❌     |✔️       |
+|Drag & Drop                             |✔️    |✔️    |✔️       |❌#306      |❌    |❌     |❌       |
+|Clipboard #162                          |❌    |❌    |❌       |❌          |❌    |❌     |❌       |
+|Raw Device Events                       |▢*#??*|▢*#??*|▢*#??*   |❌          |❌    |❌     |❌       |
+|Gamepad/Joystick events ***//DISCUSS*** |❌    |❌    |❌       |❌          |❌    |❌     |❌       |
+|Device movement events ***//DISCUSS***  |❓     |❓     |❓       |❓           |❌    |❌     |❌       |
+
+#### Details
+- **Mouse events**: Generating mouse events associated with pointer motion, click, and scrolling events.
+- **Mouse set location**: Forcibly changing the location of the pointer.
+- **Cursor grab**: Locking the cursor so it cannot exit the client area of a window.
+- **Cursor icon**: Changing the cursor icon, or hiding the cursor.
+- **Touch events**: Single-touch events.
+- **Multitouch**: Multi-touch events, including cancellation of a gesture.
+- **Keyboard events**: Properly processing keyboard events using the user-specified keymap and
+  translating keypresses into UTF-8 characters, handling dead keys and IMEs.
+- **Drag & Drop**: Dragging content into winit, detecting when content enters, drops, or if the drop is cancelled.
+  ***//DISCUSS: WINIT SUPPORTS FILE DROPS, BUT NOT TEXT OR IMAGE DROPS***
+- **Clipboard**: Winit supports copy-pasting content to and from winit.
+  ***//DISCUSS: AN ISSUE SHOULD BE OPENED REGARDING HOW THIS API SHOULD WORK***
+- **Raw Device Events**: Capturing input from input devices without any OS filtering.
+- **Gamepad/Joystick events**: Capturing input from gampads and joysticks.
+  ***//DISCUSS: SHOULD THIS BE SUPPORTED?***
+- **Device movement events:**: Capturing input from the device gyroscope and accelerometer.
+  ***//DISCUSS: SHOULD THIS BE SUPPORTED?***
```

Comment:
I suppose this is mostly relevant for iOS/Android platforms?

I'd say that if we decide Gamepad/Joystick is in scope for winit, then this should be as well.

Reply:
Yeah, and it looks like it's supported on Emscripten as well. I don't know if it works on desktop platforms, though.

I agree that it makes sense to talk about these in the same category as gamepad/joystick events.